### ISSUE TITLE
unmet peer dependencies

### ISSUE THREAD

**Issue by spacedawwwg** [nhristov/astro-compress#334]
```SSH
├─┬ @playform/compress 0.0.3
│ └─┬ @playform/build 0.0.5
│   └── ✕ unmet peer @playform/document@0.0.5: found 0.0.6 in @playform/compress
```

**Comment by NikolaRHristov** [comment]
On it.

**Comment by NikolaRHristov** [comment]
Fixed in `astro-compress` [v2.2.23](https://github.com/PlayForm/Compress/releases/tag/AstroCompress%2Fv2.2.23) and `@playform/compress` [v0.0.4](https://github.com/PlayForm/Compress/releases/tag/Compress%2Fv0.0.4)
